app/signals/apps/signals/tests/test_models.py

```diff
@@ -596,6 +596,120 @@ def test_slug_only_created_once(self):
         with self.assertRaises(ValidationError):
             category.save(slug='no-saving-me-please')
 
+    def test_parent_category_configuration(self):
+        """
+        On a parent category no additional configuration is allowed
+        """
+        parent_category = factories.ParentCategoryFactory(name='Parent category')
+        factories.CategoryFactory(name='Child category', parent=parent_category)
+
+        parent_category.configuration = None
+        parent_category.save()  # should not raise a validation error
+
+        parent_category.configuration = {'no_configuration_allowed': True}
+        with self.assertRaises(ValidationError):
+            parent_category.save()
+
+    def test_valid_show_in_filter_child_category_configuration(self):
+        """
+        On a child category only the "show_in_filter" additional configuration is allowed.
+        The "show_in_filter" should be a valid boolean value.
+        """
+        parent_category = factories.ParentCategoryFactory(name='Parent category')
+        child_category = factories.CategoryFactory(name='Child category', parent=parent_category)
+
+        child_category.configuration = {'show_in_filter': True}
+        child_category.save()
+
+        child_category.configuration = {'show_in_filter': False}
+        child_category.save()
+
+    def test_invalid_show_in_filter_child_category_configuration(self):
+        """
+        On a child category only the "show_in_filter" additional configuration is allowed.
+        The "show_in_filter" should be a valid boolean value.
+        """
+        parent_category = factories.ParentCategoryFactory(name='Parent category')
+        child_category = factories.CategoryFactory(name='Child category', parent=parent_category)
+
+        invalid_child_configurations = [
```

Comment:
This is not incorrect, but you could also parametrize this test, that way each case will show up as a separate test:
https://docs.pytest.org/en/stable/how-to/parametrize.html

Reply:
This does not work with the Django TestCase, or better the unittest TestCase, see  https://docs.pytest.org/en/latest/how-to/unittest.html#pytest-features-in-unittest-testcase-subclasses